script.kodi.windows.update/addon.xml

```diff
@@ -16,7 +16,7 @@
         <source>https://github.com/Lunatixz/KODI_Addons/tree/master/script.kodi.windows.update</source>
```

Comment:
spdx if possible

Reply:
Are we using direct .git paths now?

Reply:
it was the closest line I could find to the `<license>` element :) nothing wrong with the repo url